////////////////////////////////////////////////////////////////////////////////
/// @HdmiEdid.proto
///
/// Copyright 2018 Bose Corporation
////////////////////////////////////////////////////////////////////////////////

package DataCollectionPb;

message HdmiEdid
{
  optional string edidData = 1;
}
